nemo/src/model/rule_model/datasource.rs

```diff
@@ -215,7 +215,7 @@ impl DataSource for SparqlQuery {
 
 /// An external data source.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum DataSourceT {
+pub enum NativeDataSource {
```

Comment:
What would the non-native data sources be that we want to support?

Reply:
I'd like to build some integration to input python tuples into the database, without going through the resource provider trait (which only works for serialized data).